widgets/src/components/SummaryCard/SummaryCardContainer.js

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { MuiThemeProvider as ThemeProvider } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { DOMAINS, LOCAL } from 'api/constants';
+import theme from 'theme';
+import { getPageWidget } from 'api/app-builder/pages';
+import { getSummary } from 'api/pda/summary';
+import CustomEventContext from 'components/SummaryCard/CustomEventContext';
+import Skeleton from '@material-ui/lab/Skeleton';
+import SummaryCardValues from 'components/SummaryCard/SummaryCardValues';
+
+const styles = ({ palette }) => ({
+  root: {
+    background: '#FFF',
+    borderTop: '4px solid #E7EAEC',
+    color: '#676A6C',
+  },
+  header: {
+    display: 'flex',
+    padding: '20px 25px',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  headline: {
+    fontWeight: 'bold',
+  },
+  periodSelectRoot: {
+    border: `2px solid ${palette.primary.main}`,
+    padding: '4px 24px 4px 14px',
+    borderRadius: 4,
+    fontSize: 10,
+    fontWeight: 'bold',
+    color: palette.primary.main,
+  },
+  periodSelectIcon: {
+    right: 3,
+    color: palette.primary.main,
+  },
+  periodSelectItem: {
+    fontSize: 12,
+  },
+});
+
+class SummaryCard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      config: null,
+      loading: false,
+      loadingValues: false,
+      summary: null,
+      period: 'monthly',
+    };
+
+    this.fetchSummary = this.fetchSummary.bind(this);
+    this.fetchWidgetConfigs = this.fetchWidgetConfigs.bind(this);
+    this.handlePeriodChange = this.handlePeriodChange.bind(this);
```

Comment:
`this.handleError`

Reply:
now added it on line 68. Not sure why it didn't reflect here